src/entities/activity/ui/items/ActionPlan/Page.tsx

```diff
@@ -0,0 +1,127 @@
+  const { t } = useActionPlanTranslation();
+  const pageWidth = usePageWidth();
+  const pageMinHeight = usePageMinHeight();
+  const pageMaxHeight = usePageMaxHeight();
+  const scaledPadding = useXScaledDimension(16);
+  const scaledTopPadding = useXScaledDimension(28);
+  const scaledRightPadding = useXScaledDimension(40);
+  const scaledBottomPadding = useXScaledDimension(40);
+  const scaledLeftPadding = useXScaledDimension(36.5);
+
+  return (
+    <Box
+      sx={{
+        width: `${pageWidth}px`,
+        padding: `0 ${scaledPadding}px ${scaledPadding}px`,
+        backgroundColor: Theme.colors.light.primaryFixed,
+        borderRadius: '16px',
+      }}
+    >
+      <Title>{appletTitle}</Title>
+      <Box
+        position="relative"
+        display="flex"
+        paddingTop={`${scaledTopPadding}px`}
+        paddingRight={`${scaledRightPadding}px`}
+        paddingBottom={`${scaledBottomPadding}px`}
+        paddingLeft={`${scaledLeftPadding}px`}
+        minHeight={`${pageMinHeight}px`}
+        maxHeight={`${pageMaxHeight}px`}
+      >
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="580"
+          height="100%"
+          viewBox="0 0 580 760"
+          preserveAspectRatio="none"
+          fill="none"
+          style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            objectFit: 'fill',
+            zIndex: 1,
+          }}
+        >
+          <path
+            d="M71.5 0L9.5 29.2502L1.5 75.1427V180.04L9.5 310.657L1.5 462.96L9.5 500.783L1.5 552.223V703.013L46 741.34L158 760L271 752.435L388.5 760H479.5L527 732.263L545.5 721.672L564.5 710.073L573.5 628.374L564.5 500.783L580 298.553L573.5 153.311V29.2502L527 0H452.5H348.5L188 8.06901L71.5 0Z"
+            fill="#FCFCFF"
+          />
+          <path d="M580 760L554 677.5L502.5 718L580 760Z" fill="#FCFCFF" />
+        </svg>
+        <Box
+          position="relative"
+          display="flex"
+          flexDirection="column"
+          gap="24px"
+          width="100%"
+          height="auto"
+          overflow="hidden"
+          zIndex={2}
+        >
+          <Header>
+            {totalPages > 1
+              ? `${phrasalTemplateCardTitle} (${pageNumber}/${totalPages})`
+              : phrasalTemplateCardTitle}
+          </Header>
+          <Body>
+            {phrases.map((phrase, index) => (
+              <Phrase key={index} phrase={phrase} phrasalData={phrasalData} noImage={noImage} />
+            ))}
+          </Body>
+        </Box>
+        <Box
+          display="flex"
+          flexDirection="row"
+          alignItems="center"
+          justifyContent="center"
+          position="absolute"
+          left="0"
+          bottom="0"
+          width="100%"
+          sx={{ margin: '10px auto', zIndex: 2 }}
+        >
+          <img src={footerLogo} alt={t('credit') || ''} />
```

Comment:
Shouldn't the `10px` be a scaled value, and also explicit scaled dimensions on the `<img>`? I'm seeing this when I scale down – the logo is a lot bigger relative to the other content now:
<img width="300" src="https://github.com/user-attachments/assets/9a132d93-9e72-4271-b64c-16057585457f">